### PR TITLE
First pass at a proper README for this library

### PR DESCRIPTION
**Pull request by raar1**
Addressing #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,83 @@ To publish to the nanopub server you need to setup RSA keys. This allows the nan
...
+
+# Iterate through the provenance graph
+for s, p, o in np.provenance:
+    print(s,p,o)
+
+# See the list of concepts that are introduced by this nanopublication
+print(np.introduces_concept)
+```
+
+### Publishing an assertion as a nanopub
```
Comment (svenvanderburg) [inline README.md:72]: IMO publishing is the nicest feature, I would vote for putting this on top?

**Review by svenvanderburg — APPROVED** [review]
Wow, this is great! This will really help showcasing the library. I left some small comments.

File: README.md [hunk 2]
```diff
@@ -17,3 +17,83 @@ To publish to the nanopub server you need to setup RSA keys. This allows the nan
...
+# Iterate through the publication info
+for s, p, o in np.pubinfo:
+    print(s, p, o)
+
+# Iterate through the provenance graph
+for s, p, o in np.provenance:
+    print(s,p,o)
+
+# See the list of concepts that are introduced by this nanopublication
+print(np.introduces_concept)
```
[the remark this answers is not in the document]
Comment (raar1) [reply README.md:69]: Oh no, yes, sorry. I got confused. I think at some point we discussed having more than one concept. Possibly Tobias said it wouldn't make sense - that might be why we dropped it. I'll change it now.

File: README.md [hunk 3]
```diff
@@ -17,3 +17,83 @@ To publish to the nanopub server you need to setup RSA keys. This allows the nan
...
 ```
+
+## Quick Start
+
+### Searching for nanopublications
+```python
+from nanopub import NanopubClient
+
+# Create the client, that allows searching, fetching and publishing nanopubs
+npclient = NanopubClient()
```
[the remark this answers is not in the document]
Comment (raar1) [reply README.md:28]: Agreed, and done.

File: README.md (hunk 1 above)
Comment (raar1) [reply README.md:72]: Done!

**Pull request merged**